workflow/envs/bedtools.yaml

```diff
@@ -0,0 +1,7 @@
+---
+channels:
+  - conda-forge
+  - bioconda
```

Comment:
Some updates:

- `cutadapt`, `gffutils`, `pysam`, `R` and `samtools` need to use both channels with `conda-forge` the first otherwise a dependency error raises.
- `oligmap`  needs to use both channels but `bioconda` can go first.